src/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './MovieCard.css';
+
+const MovieCard = ( {image, title, rating}) => {
+  return (
+    <section className="movie-card">
+      <div className="image-holder">
+        <img className="card-image" src={image}/>
+      </div>
+      <h3 className="card-title">Movie Title: {title}</h3>
+      <h3 className="rating">Movie Rating: {Math.round( rating * 10 ) / 10}</h3>
```

Comment:
I love that we can put a method like this in the JSX! Learning so much! This is fun!